typings/balena-register-device.d.ts

```diff
@@ -1,6 +1,14 @@
 // Typings (incomplete) for balena-register-device@v6.0.1
+
 // TODO: Upstream types to the repo
 declare module 'balena-register-device' {
+	import TypedError = require('typed-error');
+	import { Response } from 'request';
+
+	factory.ApiError = class ApiError extends TypedError {
+		response: Response;
+	};
+
 	function factory({
```

Comment:
It'd be better to put this avoid the `factory.ApiError =` just to avoid confusing people

Reply:
I'm not sure what you mean by this?

Reply:
The first `avoid` was meant to be `above` - but you did it anyway so it's all cool